styles/ansi.less

```diff
@@ -0,0 +1,162 @@
+@import "ui-variables";
+
+.go-plus-ansi-bold {
+  font-weight: bold;
+}
+
+.go-plus-ansi-dim {
+  font-weight: lighter;
+}
+
+.go-plus-ansi-italic {
+  font-style: italic;
+}
+
+.go-plus-ansi-underline {
+  text-decoration: underline;
+}
+
+.go-plus-ansi-inverse {
+  color: @input-background-color;
+  background-color: @text-color;
+}
+
+.go-plus-ansi-hidden {
+  color: transparent;
+}
+
+.go-plus-ansi-strikethrough {
+  text-decoration: line-through;
+}
+
+// TODO(zb): offer a setting for different color palette(s) or maybe hook into syntax theme
+
+// Text Colors
+// -----------
+
+.go-plus-ansi-black {
+    color: black;
+}
+
+.go-plus-ansi-green {
+  color: green;
+}
+
+.go-plus-ansi-yellow {
+  color: yellow;
+}
+
+.go-plus-ansi-blue {
+  color: blue;
+}
+
+.go-plus-ansi-magenta {
+  color: magenta;
+}
+
+.go-plus-ansi-cyan {
+  color: cyan;
+}
+
+.go-plus-ansi-white {
+  color: white;
+}
+
+.go-plus-ansi-gray {
+  color: gray;
+}
+
+.go-plus-ansi-red {
+  color: red;
+}
+
+.go-plus-ansi-lightRed {
+  color: lighten(red, 20%);
+}
+
+.go-plus-ansi-lightGreen {
+  color: lighten(green, 20%);
+}
+
+.go-plus-ansi-lightYellow {
+  color: lighten(yellow, 20%);
+}
+
+.go-plus-ansi-lightBlue {
+  color: lighten(blue, 20%);
+}
+
+.go-plus-ansi-lightMagenta {
+  color: lighten(magenta, 20%);
+}
+
+.go-plus-ansi-lightCyan {
+  color: lighten(cyan, 20%);
+}
+
+.go-plus-ansi-lightWhite {
+  color: lighten(white, 20%);
```

Comment:
light white -> whiter than white 😆
maybe set normal white to a slightly greyish color and light white to white